src/pages/Home/index.tsx

```diff
@@ -1,14 +1,40 @@
 import { Container } from 'react-bootstrap';
-
+import { useEffect, useState } from 'react';
+import { OrderProps, BobaProps } from 'types/common/main';
+import { bobaList } from 'assets/sampleBobaAPI';
 import BobaContainer from 'components/BobaContainer';
-import { OrderProps } from 'types/common/main';
+import SearchBar from 'components/SearchBar';
 
 const Homepage = (props: OrderProps): JSX.Element => {
   const { order, setOrder } = props;
+
+  const [bobaListx, setBobaListx] = useState<BobaProps[]>(bobaList);
```

Comment:
Rename the state variable bobaListx to something more descriptive, such as filteredBobaList:
```
const [filteredBobaList, setFilteredBobaList] = useState<BobaProps[]>(bobaList);
```